core/src/main/java/gov/nasa/jpl/mudrod/ssearch/ClickstreamImporter.java

```diff
@@ -78,7 +80,7 @@ public void importfromCSVtoES() {
         String[] clicks = line.split(cvsSplitBy);
         for (int i = 1; i < clicks.length; i++) {
           if (!clicks[i].equals("0.0")) {
-            IndexRequest ir = new IndexRequest(props.getProperty("indexName"), props.getProperty("clickstreamMatrixType"))
+            IndexRequest ir = new IndexRequest(props.getProperty("indexName"), props.getProperty(MudrodConstants.CLICK_STREAM_MATRIX_TYPE))
```

Comment:
we should change props.getProperty("indexName") to props.getProperty(MudrodConstants.INDEX_NAME) in this line